tablespace/benchmark.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+wget --no-verbose --continue 'https://datasets.clickhouse.com/hits_compatible/hits.tsv.gz'
+guzip hits.tsv.gz
+chmod 666 hits.tsv
+
+psql 'host=<hostname> port=5432 dbname=csdb user=csuser password=<password> sslmode=require' < create.sql
+
+psql 'host=<hostname> port=5432 dbname=csdb user=csuser password=<password> sslmode=require' -t -c '\timing' -c "\\copy hits FROM 'hits.tsv'"
```

Comment:
I tried as per the instruction but this COPY was stuck for more than an hour.

Reply:
@chhetripradeep What region is your client VM running from? Also did the `hits` table get populated with any tuples at all or was it empty when you killed the COPY?

Reply:
I was doing from my local machine in Singapore. I will give it a try again with client in same region as server.

Reply:
Yes please try one closer. The load from a client in the same region takes ~30+ mins. It’s not infeasible for it to take longer if your client VM is located far away.  The README instructions suggest users to use a client VM in East US. That’s where the DBs are located.

Reply:
Hi @osawyerr 
I ran it again from us-east-1 in aws, but i think since the tablespace instance is in oracle cloud, still the COPY command took around 1.5 hours for me. I am going to create a oracle cloud account for client and rerun the test today.